.github/workflows/hrm-lambda-deploy.yml

```diff
@@ -0,0 +1,150 @@
+name: 'Deploy HRM Lambda'
```

Comment:
Would've been nice if this could have been moved rather than deleted/created but I appreciate that's not always possible

Reply:
Sorry about that. I have a terrible habit of moving things at the OS level and crossing my fingers that git will pick up the similarities and mark them as moved. I realized after the commit that this hadn't been recognized as a move. I'll try to do a better job of using `git mv`